### DISCUSSION TITLE
DataGrid block certain columns from being resizeable

### DISCUSSION THREAD

**Discussion by bastigerner** [Megabit/Blazorise#3142] [Q&A]
Hey,

short question:
We have set out datagrid to be resizeable.
But we have certain columns which should not be resizeable.
Is it possible to set this option on those?

Kind regards

**Comment by stsrki** [comment]
It is not possible at the moment. But it could be a good feature to have. @David-Moreira ?

**Reply by bastigerner** [reply]
Okay thought so didnt find anything. But yes sure would be a much wanted feature from our side.
